README.md

```diff
@@ -80,33 +80,72 @@ metric exporters, service monitors, and custom dashboards.
+- creating the location for your local customization files
+- using the `USER_DIR` environment variable to specify the location of the customization files
+- specifying customization variables and parameters in the customization files
 
-The components that are deployed are highly customizable. The recommended
-approach for customization is to set a `USER_DIR` environment variable. `USER_DIR`
-points to a directory containing user-modified customization files which can exist
-outside of your repository. You can use customization files to check in
-environment-specific customizations into a separate Git repository, for example.
+### Creating the Location for Customization Files
 
-`USER_DIR` must refer to a directory. The directory can include any or all of
-the following files in the structure below. Additional content is allowed, but
-will be ignored.
+The `USER_DIR` environment variable enables you to use a directory outside of the local repository to contain the customization files. By using a location outside of the repository, you can put the customization files in a location that is under version control, and you can preserve your customizations when you deploy new versions of the monitoring and logging applications. You could also create different sets of configuration files for different cluster types, and then use the `USER_DIR` variable to specify the configuration files to use based on the cluster type on which you were deploying.
+
+For example, to create a customization directory named *my-viya4mon-user-dir* that contains directories for monitoring and logging customization files:
+
+```bash
+mkdir -p ~/my-viya4mon-user-dir/monitoring
+mkdir -p ~/my-viya4mon-user-dir/logging
+```
+
+Note that these commands create directories under your root directory.
+
+### Using USER_DIR to Specify the Location of Customization Files
+
+Use the `USER_DIR` environment variable to specify the root location of your local customization files.
+
+This example sets the `USER_DIR` environment variable to the *my-viya4mon-user-dir* directory:
+
+```bash
+export USER_DIR=~/my-viya4mon-user-dir
+```
+The monitoring and logging deployment scripts use the customization files contained in the directories user the `USER_DIR` location.
+
+### <a name="customization"></a>Specifying Customization Variables and Parameters
+
+After you create the location for your customization files, you can customize the components that you deploy for monitoring and logging by specifying environment variables and Helm chart parameters in this set of customization files (in this example, contained in the *my-viya4mon-user-dir* root directory):
 
 ```text
-- user.env
-- [monitoring]
-  - user.env
-  - user-values-prom-operator.yaml
-  - user-values-pushgateway.yaml
-- [logging]
-  - user.env
-  - user-values-elasticsearch-open.yaml
-  - user-values-es-exporter.yaml
-  - user-values-fluent-bit-open.yaml
+/my-viya4mon-user-dir/user.env
```

Comment:
It's a nit, but could you remove the leading / in these paths? That gives a better indication that `my-viya4mon-user-dir` can be located anywhere.

Note that the path appears in several places across several README.md files. I didn't add a comment for each occurrence.

Reply:
Changed